test/helpers/TestParachain.sol

```diff
@@ -24,18 +24,17 @@ contract TestParachain is Parachain {
         reportStakeWithdrawRequested(_parachain, _account, _amount, _staker);
     }
 
-    function reportSlashExternal(
+    function reportSlashExternal(IRegistry.Parachain memory _parachain, bytes memory _reporter, uint256 _amount)
+        external
+    {
+        reportSlash(_parachain, abi.encodePacked(_reporter), _amount);
```

Comment:
Not sure of the specifics here, but does it need the _reporter parameter to be abi encoded as it is already in bytes?

Reply:
Thanks for spotting. Fixed in latest commit.